docs/access.md

```diff
@@ -1,20 +1,118 @@
 # Access management
 
-## LaminDB
+This document outlines the access control system, detailing its architecture, key features, roles, and implementation guidelines.
 
-LaminDB doesn't come with access management but relies on the underlying data infrastructure for this.
+## System architecture
 
-## LaminHub
+Our access management is built on a four-tier hierarchy, providing granular control over resources:
 
-```{include} includes/access-laminhub.md
+1.  **Organizations:** The top-level entity. Contains all associated teams, instances, and spaces. Acts as the primary boundary for users and resources.
+2.  **Teams:** Groups of users within an organization. Roles and permissions can be assigned collectively to teams, simplifying management.
+3.  **Instances:** Collaborative workspaces (e.g., projects, environments) within an organization. Each instance has its own permission structure and collaborators.
```

Comment:
> Collaborative workspaces (e.g., projects, environments)

This is quite inaccurate because there are many differences between a workspace as coined by the SaaS industry and a LaminDB instance. A much better comparison is a git repository.

Rather: LaminDB instances are the central databases at the heart of the data platform.